src/annotation/myannotation/AdulthoodAnnotationProcessor.java

```diff
@@ -0,0 +1,51 @@
+package annotation.myannotation;
+
+import annotation.exception.ExpectedTypeException;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.time.Period;
+
+public class AdulthoodAnnotationProcessor {
```

Comment:
Please create an AnnotationProcessor interface with the process method then implement all your processors from that one